.github/ISSUE_TEMPLATE/brand_new_feature.yml

```
@@ -0,0 +1,26 @@
+name: Requesting New Feature
+description: If you are adding something completely new then use this template!
+title: "New Feature Request: <title>"
```

Redundant with labels
```suggestion
title: "<title>"
```